packages/playwright/src/takeSnapshot.ts

```diff
@@ -35,7 +35,15 @@ async function takeSnapshot(
   // Serialize and capture the DOM
   const domSnapshot: elementNode = await page.evaluate(dedent`
     ${rrweb};
-    rrwebSnapshot.snapshot(document);
+    if (typeof define === "function" && define.amd) {
```

Comment:
It'd be great to have a comment here that elaborates on this supporting AMD and why it's needed.